Eks (#454)
tigera/docs: calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

   kubectl apply -f {{manifestsUrl}}/manifests/calico-vxlan.yaml
   ```

> 2. Configure {{prodname}} to disable AWS src/dst checks.

Reviewer: Do the numbers here need to be changed manually? I thought we always put `1` and liquid handles the explicit numbering for us?

Reviewer 2: There are circumstances where they need to be explicit, yes. I'm not sure of the exact cause. But having multiple paragraphs for a single step, particularly with different markup, such as that for notes and codeblocks, significantly increases the likelihood that automatic numbering will fail.

   ```batch
   kubectl -n kube-system set env daemonset/calico-node FELIX_AWSSRCDSTCHECK=Disable
   ```

</TabItem>

> <TabItem label="Helm" value="Helm-2">

Reviewer: I might be misunderstanding what this does, but I'm kind of confused why we have tabs here for helm when I don't see any other tabs. Are there other tabs that I'm missing?

Reviewer: Nvm, I see that I misread things. This should be fine.

Reviewer 3: Although already introduced by previous author, using inline tabs inside a procedure has issues. The final step for all tabs comes out as "1." in netlify. I see two options. Duplicate the content for each tab (which is the original intent of tabs), or make the final step to be a non-numbered step. Like this:

Finally, add nodes to the cluster.

<img width="796" alt="image" src="https://user-images.githubusercontent.com/48458529/224825181-75ca5d55-eb01-4e76-8d82-ad9d0cc62368.png">

   kubectl -n kube-system set env daemonset/calico-node FELIX_AWSSRCDSTCHECK=Disable
   ```

</TabItem>

<TabItem label="Helm" value="Helm-2">

1. Add {{prodname}} into your Helm repository.

    ```batch
    helm repo add projectcalico https://docs.projectcalico.org/charts
    ```

> 1. If you already have {{prodname}} added, you can update it to get the latest released version.

Reviewer 3: ```suggestion
1. If {{prodname}} is already added, update it to get the latest released version.
```
